ocaml/xenopsd/xc/domain.ml

```diff
@@ -2000,3 +2000,10 @@ let move_xstree ~xs domid olduuid newuuid =
     )
   in
   List.iter mv_tree search_paths
+
+(** Request a fast_resume, return without waiting for acknowledgement *)
+let fast_resume ~xc ~xs domid =
+  let uuid = get_uuid ~xc domid in
+  debug "VM = %s; domid = %d; Requesting domain fast resume"
+    (Uuid.to_string uuid) domid ;
+  Xenctrl.domain_resume_fast xc domid
```

Comment:
We might want to check whether the guest supports fast resume or not before attempting this, see the comments in Xen's `xg_resume.c` (Unfortunately it doesn't document how it can be determined whether a guest supports this or not, but presumably it can advertise that in xenstore in some way? although the `xl` doesn't seem to check for it either). Perhaps just add a TODO comment till we figure it out?